<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lâm Thuỳ Dung</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/ic_array.png</normaloff>../images/ic_array.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 240, 220);
color: rgb(19, 16, 16);</string>
    </property>
    <property name="title">
     <string>Mảng gốc:</string>
    </property>
    <widget class="QLineEdit" name="lineEditMangGoc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(240, 187, 120);
color: rgb(19, 16, 16);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 240, 220);
color: rgb(19, 16, 16);</string>
    </property>
    <property name="title">
     <string>Kết quả:</string>
    </property>
    <widget class="QLineEdit" name="lineEditKetQua">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>#9Slide03 Arima Madurai ExtraBo</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(240, 187, 120);
color: rgb(19, 16, 16);</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonXuatMangNgauNhien">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 240, 220);
color: rgb(19, 16, 16);</string>
    </property>
    <property name="text">
     <string>Xuất mảng ngẫu nhiên</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTimPTNhoNhat">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(240, 187, 120);
color: rgb(19, 16, 16);</string>
    </property>
    <property name="text">
     <string>Tìm phần tử nhỏ nhất</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTinhTongMang">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(240, 187, 120);
color: rgb(19, 16, 16);</string>
    </property>
    <property name="text">
     <string>Tính tổng giá trị mảng</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTangPT2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 240, 220);
color: rgb(19, 16, 16);</string>
    </property>
    <property name="text">
     <string>Tăng mỗi ph.tử mảng lên 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSXMangGiamDan">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 240, 220);
color: rgb(19, 16, 16);</string>
    </property>
    <property name="text">
     <string>Sắp xếp mảng giảm dần</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSXMangTangDan">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(240, 187, 120);
color: rgb(19, 16, 16);</string>
    </property>
    <property name="text">
     <string>Sắp xếp mảng tăng dần</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTongPTLe">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(240, 187, 120);
color: rgb(19, 16, 16);</string>
    </property>
    <property name="text">
     <string>Tổng giá trị phần tử lẻ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDemPTLe">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 240, 220);
color: rgb(19, 16, 16);</string>
    </property>
    <property name="text">
     <string>Đếm số phần tử lẻ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonThoat">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>460</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>#9Slide03 Arima Madurai ExtraBo</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(255, 240, 220);
color: rgb(19, 16, 16);</string>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/ic_thoat.png</normaloff>../images/ic_thoat.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
